src/node_util.cc

```diff
@@ -356,6 +356,16 @@ static void ToUSVString(const FunctionCallbackInfo<Value>& args) {
                              value.length()).ToLocalChecked());
 }
 
+static void Ref(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  env->add_refs(1);
```

Comment:
I'm unsure if it's appropriate to use `add_refs()` for this, that's for task queues (i.e., `worker_threads`, not `cluster`.)

cc @addaleax - or is it?

Reply:
I don't think it's impossible to use it this way, but yeah, this isn't really what I would expect this to be used for. Ideally, there should be an actual handle that could be ref'ed/unref'ed instead...

Reply:
I was thinking that too, but of course it isn't backed by a real I/O object. I suppose you could fake it with an eternal timer (I phrase I just made up to mean a repeating timer with a long interval.)